HLTrigger/Configuration/python/HLT_75e33/modules/ticlTrackstersMergeV3_cfi.py

```diff
@@ -0,0 +1,36 @@
+import FWCore.ParameterSet.Config as cms
+
+ticlTrackstersMerge = cms.EDProducer("TrackstersMergeProducerV3",
```

Comment:
I am confused about the usage of this producer. It doesn't seem to be referenced in the menu.
What's the purpose of this configuration fragment?

Reply:
Ciao @mmusich you are right and that will be cleaned/removed in the (near) future.